PagingMenuController.podspec

```diff
@@ -9,13 +9,13 @@
 
 Pod::Spec.new do |s|
   s.name             = "PagingMenuController"
-  s.version          = "0.7.9"
+  s.version          = "0.7.10b"
   s.summary          = "A paging view with customizable menu"
   s.homepage         = "https://github.com/kitasuke/PagingMenuController"
   # s.screenshots     = "www.example.com/screenshots_1", "www.example.com/screenshots_2"
   s.license          = 'MIT'
   s.author           = { "kitasuke" => "yusuke2759@gmail.com" }
-  s.source           = { :git => "https://github.com/kitasuke/PagingMenuController.git", :tag => s.version.to_s }
+  s.source           = { :git => "https://github.com/HorizonteNovasMidias/PagingMenuController.git", :tag => s.version.to_s }
```

Comment:
Sorry about the lack of attention. The private podspec was due to the urgency of the fixed bugs and we now use the fork in a private repository